spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -199,4 +204,41 @@ public void testImportAsHiveTable() throws Exception {
     long count2 = spark.read().format("iceberg").load(DB_NAME + ".test_partitioned_table").count();
     Assert.assertEquals("three values ", 3, count2);
   }
+
+  @Test
+  public void testImportWithIncompatibleSchema() throws Exception {
```

Comment:
This should be named `testImportWithNameMapping` instead. The schema isn't incompatible, it just uses a different strategy (name mapping, instead of position mapping) to assign IDs for fields.

Reply:
Done.